.env.example

```
@@ -1,5 +1,24 @@
-PORT=3333
-HOST=0.0.0.0
-NODE_ENV=development
-APP_KEY=73rHnSEgVPLfFLpTk8omwzBrMk-c75rX
-DRIVE_DISK=local
+# Application Environment Variables
+NODE_ENV=
+PORT=
+API_URL=""
+FRONTEND_URL=""
+DATABASE_URL=""
+APP_SECRET=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The APP_SECRET key should go before the DATABASE_URL key

(UnorderedKey)